packages/edit-post/src/components/header/header-toolbar/index.native.js

```diff
@@ -43,8 +43,14 @@ function HeaderToolbar( {
 	noContentSelected,
 } ) {
 	const wasNoContentSelected = useRef( noContentSelected );
+	// eslint-disable-next-line no-unused-vars
 	const [ isInserterOpen, setIsInserterOpen ] = useState( false );
```

Comment:
Although `isInserterOpen` and other references to the open inserter are no longer used in this file, `isInserterOpen` is still used in the media block insertion PR. Rather than remove all of the references here, `// eslint-disable-next-line no-unused-vars` is used. This line can be removed once merged.

Reply:
I've removed this code in https://github.com/WordPress/gutenberg/pull/52269/commits/3b4c5c2f2406a6879bce9a43f030de4f1eade8e0 👍